vmdb/app/helpers/application_helper.rb

```diff
@@ -1758,7 +1758,7 @@ def set_edit_timer_from_schedule(schedule)
       @edit[:new][:timer_weeks]  = schedule.run_at[:interval][:value] if schedule.run_at[:interval][:unit] == "weekly"
       @edit[:new][:timer_days]   = schedule.run_at[:interval][:value] if schedule.run_at[:interval][:unit] == "daily"
       @edit[:new][:timer_hours]  = schedule.run_at[:interval][:value] if schedule.run_at[:interval][:unit] == "hourly"
-      t                          = schedule.run_at[:start_time].to_time(:utc).in_time_zone(@edit[:tz])
+      t                          = schedule.run_at[:start_time].utc.in_time_zone(@edit[:tz])
```

Comment:
> Can we do something about the fact that schedule.run_at[:start_time] is sometimes a String and sometimes a Time? It seems that would make this simpler.

Yes, thats precisely what I'm trying to address now.
@matthewd Does this look reasonable to you ?
```
schedule.run_at[:start_time].to_s.to_time(:utc).in_time_zone(@edit[:tz])
```
There are 2 things that are addressed here -
1. Ensured that ```schedule.run_at[:start_time]``` is always converted to a ```String```, that way it is at par with the spec
2. Calling ```.to_time``` with parameter ```:utc``` addresses the Rails 4 deprecation warning that I had seen earlier when ```.to_time``` was called without parameters
```
DEPRECATION WARNING: Rails 4 changes the default of String#to_time to local.  Please pass the type of conversion you want, like to_time(:utc) or to_time(:local).
```